worker/src/durable.rs

```diff
@@ -200,7 +211,9 @@ pub struct Storage {
 
 impl Storage {
     /// Retrieves the value associated with the given key. The type of the returned value will be
-    /// whatever was previously written for the key, or undefined if the key does not exist.
+    /// whatever was previously written for the key.
+    ///
+    /// Returns [Err] if the key does not exist.
```

Comment:
I got tripped up by this while testing

Reply:
I guess we accidentally left this in when copy pasting the JS documentation, I'll deal with that in a follow up PR when I clean  up some documentation.